WtWSMS/common/landed_titles/BP_e_britannia.txt

```diff
@@ -2955,71 +3651,57 @@ e_britannia = {
+					name_list_romano_noric = cn_praesidium
+					name_list_romano_gothic = cn_praesidium
+					name_list_romano_dacian = cn_praesidium
+					name_list_romano_frank = cn_praesidium
+					name_list_romano_aramean = cn_praesidium
+					name_list_romano_vandalic = cn_praesidium
+					#Celtic Cultures
+					name_list_briton = cn_gwycgryw
 				}
 				b_warwick = {
 					province = 1603
 					cultural_names = {
 						#Roman Cultures
-						name_list_roman = cn_praesidium_cornaviorum
-						name_list_romano_gallic = cn_praesidium_cornaviorum
-						name_list_romano_hispanic = cn_praesidium_cornaviorum
-						name_list_romano_british = cn_praesidium_cornaviorum
-						name_list_romano_african = cn_praesidium_cornaviorum
-						name_list_romano_raetian = cn_praesidium_cornaviorum
-						name_list_romano_thracian = cn_praesidium_cornaviorum
-						name_list_romano_illyrian = cn_praesidium_cornaviorum
-						name_list_romano_pannonian = cn_praesidium_cornaviorum
-						name_list_dalmatian = cn_praesidium_cornaviorum
-						name_list_romano_noric = cn_praesidium_cornaviorum
-						name_list_romano_gothic = cn_praesidium_cornaviorum
-						name_list_romano_dacian = cn_praesidium_cornaviorum
-						name_list_romano_frank = cn_praesidium_cornaviorum
-						name_list_romano_aramean = cn_praesidium_cornaviorum
-						name_list_romano_vandalic = cn_praesidium_cornaviorum
+						name_list_roman = cn_praesidium
+						name_list_romano_gallic = cn_praesidium
+						name_list_romano_hispanic = cn_praesidium
+						name_list_romano_british = cn_praesidium
+						name_list_romano_african = cn_praesidium
+						name_list_romano_raetian = cn_praesidium
+						name_list_romano_thracian = cn_praesidium
+						name_list_romano_illyrian = cn_praesidium
+						name_list_romano_pannonian = cn_praesidium
+						name_list_dalmatian = cn_praesidium
+						name_list_romano_noric = cn_praesidium
+						name_list_romano_gothic = cn_praesidium
+						name_list_romano_dacian = cn_praesidium
+						name_list_romano_frank = cn_praesidium
+						name_list_romano_aramean = cn_praesidium
+						name_list_romano_vandalic = cn_praesidium
+						#Celtic Cultures
+						name_list_briton = cn_gwycgryw
 					}
 					color = { 190 30 30 }
 				}
 				b_coventry = {
 					province = 1604
 					cultural_names = {
-						#Roman Cultures
-						name_list_roman = cn_lunt
-						name_list_romano_gallic = cn_lunt
-						name_list_romano_hispanic = cn_lunt
-						name_list_romano_british = cn_lunt
-						name_list_romano_african = cn_lunt
-						name_list_romano_raetian = cn_lunt
-						name_list_romano_thracian = cn_lunt
-						name_list_romano_illyrian = cn_lunt
-						name_list_romano_pannonian = cn_lunt
-						name_list_dalmatian = cn_lunt
-						name_list_romano_noric = cn_lunt
-						name_list_romano_gothic = cn_lunt
-						name_list_romano_dacian = cn_lunt
-						name_list_romano_frank = cn_lunt
-						name_list_romano_aramean = cn_lunt
-						name_list_romano_vandalic = cn_lunt
```

Comment:
Why the removal?

Reply:
> Why the removal?

Because it was completely nonsensical, just like Coventry Lunt is an Anglo-Saxon placename. We need to find something Briton or Latin instead.

Reply:
It refers to this: https://en.wikipedia.org/wiki/Lunt_Roman_Fort

Reply:
> It refers to this: https://en.wikipedia.org/wiki/Lunt_Roman_Fort

I know about this fort, but its Roman name is unknown. Lunt is the modern name of Anglo-Saxon origin.

Reply:
Well if you have an alternative name or want to Latinise it go ahead. But removing a reference to an attested fort because the name is Anglo-Saxo seems like the wrong solution to the problem

Reply:
> Well if you have an alternative name or want to Latinise it go ahead. But removing a reference to an attested fort because the name is Anglo-Saxo seems like the wrong solution to the problem

How about Conventranum from Orbis latinus? This is one of the later Latin names for Coventry, but it doesn't sound very Germanic.

Reply:
Ok, we can go with Conventranum then